### PR TITLE
Server grails

### PR DESCRIPTION
**Pull request by demoth**
Basic functionality to create request and approve debts (in memory db)

### CODE REVIEW

File: server/grails-app/domain/com/dao/mydebts/server/Debt.groovy [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        this.src = json.src
+        this.dest = json.dest
+        this.comment = json.comment
+        this.amount = json.amount as BigDecimal
+        this.approvedBySrc = json.approvedBySrc as Boolean
+        this.approvedByDest = json.approvedByDest as Boolean
+    }
+
+    @Override
+    public String toString() {
```
Comment (Adonai) [inline server/grails-app/domain/com/dao/mydebts/server/Debt.groovy]: Can't we use `@Canonical` here?

**Comment by Adonai** [comment]
There are many files that are related to human-viewable endpoints of application (e.g. htmls, js, css, even for mobile devices!)

Do we need such level of representation? Can we get rid of all it and leave only API endpoints that operate with JSON?

File: server/grails-app/init/server/Application.groovy [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package server
+
+import grails.boot.GrailsApp
+import grails.boot.config.GrailsAutoConfiguration
+
+class Application extends GrailsAutoConfiguration {
+    static void main(String[] args) {
+        GrailsApp.run(Application, args)
+    }
+}
```
Comment (asmolko) [inline server/grails-app/init/server/Application.groovy]: missing eol

Comment (demoth) [inline server/grails-app/init/server/Application.groovy]: added

File: server/grails-app/conf/application.yml [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+grails:
+    profile: web
+    codegen:
+        defaultPackage: server
+    spring:
+        transactionManagement:
+            proxies: false
+info:
+    app:
+        name: '@info.app.name@'
```
Comment (Adonai) [inline server/grails-app/conf/application.yml]: MyDebts, 1.0, 3.5.9

File: server/grails-app/domain/com/dao/mydebts/server/Debt.groovy [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    static mapping = {
+        id generator: 'hilo',
+                params: [table: 'hi_value', column: 'next_value', max_lo: 100]
+    }
+
+    Debt() {
+    }
+
+    @Override
+    public String toString() {
```
Comment (Adonai) [inline server/grails-app/domain/com/dao/mydebts/server/Debt.groovy]: Can't we mark class as `@ToString` ?

**Comment by demoth** [comment]
this is going to die

**Comment by Adonai** [comment]
So what, declining is fine I suppose?

**Pull request closed without merging**